spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/geometries/SCSpatialFeature.java

```diff
@@ -130,4 +130,16 @@ public String toJson() {
         }
         return null;
     }
+
+    public  Map<String, Object> toMap() {
+        Map<String, Object> map = null;
+        try {
+            map = SCObjectMapper.getMapper().convertValue(this,  Map.class);
+            map.put("type", "Feature");
+        } catch (Exception e) {
+            String s = "";
```

Comment:
Can you log the exception as a warning here?  Also, `String s = "";` doesn't appear to be doing anything